tests/unit/FormTextArea.spec.js

```diff
@@ -100,26 +112,32 @@ describe('FormTextArea', () => {
     const errorText = 'This field has an error';
     const wrapper = factory({
       name: 'FormTextAreaField',
-      error: errorText,
-      validation: 'required'
     });
-    
+
+    wrapper.setProps({value: "", validation: 'required'});
+    expect(wrapper.find('.is-invalid').exists()).toBe(true);
     expect(wrapper.find('textarea').classes('is-invalid')).toBe(true);
     expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
     expect(wrapper.find('.invalid-feedback').text()).toContain(requiredText);
+
+    wrapper.setProps({value: "text", error: errorText});
+    expect(wrapper.find('.is-invalid').exists()).toBe(true);
+    expect(wrapper.find('textarea').classes('is-invalid')).toBe(true);
+    expect(wrapper.find('.invalid-feedback').isVisible()).toBe(true);
     expect(wrapper.find('.invalid-feedback').text()).toContain(errorText);
   });
 
   it('removes the validation error messages when the field is valid.', () => {
     const errorText = 'This field has an error';
     const wrapper = factory({
       name: 'FormTextAreaField',
-      error: errorText,
-      validation: 'required'
     });
-    
-    wrapper.setProps({value: "Hello World"});
-    
+
+    wrapper.setProps({value: '', validation: 'required'});
+    expect(wrapper.find('.invalid-feedback').exists()).toBe(true);
+    expect(wrapper.find('.invalid-feedback').text()).toContain("The FormTextAreaField field is required.");
```

Comment:
The `FormTextArea` component contains two elements with the `invalid-feedback` class. I think you need to remove this ambiguity. The `find` method finds the 1st occurrence of the matched element, and since there are two, this test could fail unexpectedly in the future if things change slightly.

Reply:
I changed it by creating a component to show the errors